TODOs in Lecture 2 (#3)
adamtopaz/UAlberta-412-512-lecture-notes: lecture02.tex

Recall that the algebra $B$ is said to be \emph{of finite type} provided that $B$ is finitely generated, meaning that $B = A[b_{1},\ldots,b_{n}]$ for some choice of $b_{1},\ldots,b_{n} \in B$.
A stronger notion of finiteness is being a \emph{finite} algebra, which means that $B$ is finitely generates as an $A$-module.

> \begin{lemma}[Transitivity of Finiteness]\label{lemma:transitivity of finiteness}
> 	Let $A$ be a finitely generated $B$-module and $B$ a finitely generated $C$-module. Then $A$ is finitely generated as a $C$-module. 
> \end{lemma}

Reviewer: Please formulate the lemmas as follows.
```suggestion
\begin{lemma}[Transitivity of Finiteness]\label{lemma:transitivity of finiteness}
  Let $B$ be a finite $A$-algebra and let $C$ be a finitely-generated $B$-module.
  Then $C$ is a finitely-generated $A$-module.
\end{lemma}
```
Also, please use 2 spaces instead of tabs, and ensure that each sentence is on it's own line.

\begin{proof}

> 	Suppose $\{a_i\colon i= 1,\cdots, m\}$ is a generator of $A$ as a $B$-module and $\{b_i\colon i= 1,\cdots, n\}$ is a generator of $B$ as a $C$-module. Then for any $b\in B$, there exist $c_i(b)\in C$ for $i = 1,\cdots, n$ such that $b = \sum_i c_i(b)b_i$. Next, for an arbitrary $a\in A$, there exist $b_j'\in B$ for $j = 1, \cdots, m$ such that \[a = \sum_{j = 1}^m b_j' a_j = \sum_{j = 1}^m \left(\sum_{i=1}^n c_i(b_j')\cdot b_i\right)\cdot a_j = \sum_{j = 1}^m \sum_{i=1}^n c_i(b_j')\cdot b_ia_j,\]

Reviewer: Replace `generator` by `system of generators`. 
Also, please modify the proof so that the notation matches the modified statement of the lemma.

> 	Suppose $\{a_i\colon i= 1,\cdots, m\}$ is a generator of $A$ as a $B$-module and $\{b_i\colon i= 1,\cdots, n\}$ is a generator of $B$ as a $C$-module. Then for any $b\in B$, there exist $c_i(b)\in C$ for $i = 1,\cdots, n$ such that $b = \sum_i c_i(b)b_i$. Next, for an arbitrary $a\in A$, there exist $b_j'\in B$ for $j = 1, \cdots, m$ such that \[a = \sum_{j = 1}^m b_j' a_j = \sum_{j = 1}^m \left(\sum_{i=1}^n c_i(b_j')\cdot b_i\right)\cdot a_j = \sum_{j = 1}^m \sum_{i=1}^n c_i(b_j')\cdot b_ia_j,\]

Reviewer: Why did you choose the notation `c_i(b)`?

  Argue by induction on $n$.
  The case $n = 0$ is trivial.
  In general, note that $A[b_{1},\ldots,b_{n+1}] = A[b_{1},\ldots,b_{n}][b_{n+1}]$.

>   By induction $A[b_{1},\ldots,b_{n}]$ is finite over $A$, so by Lemma \ref{lemma:transitivity of finiteness} it sufficies to assume $n = 1$, and put $b = b_{1}$.

Reviewer: ```suggestion
  By induction $A[b_{1},\ldots,b_{n}]$ is finite over $A$, so by Lemma~\ref{lemma:transitivity of finiteness} it suffices to assume $n = 1$, and put $b = b_{1}$.
```

  In this case, let $g$ be a nonzero monic polynomial over $A$ such that $g(b) = 0$.

>   Note that $A[b]$ is a quotient of $A[X]/(g)$, which is easily seen to be finitely-generated as a module over $A$, as it is generated by the images of $1,X,\ldots,X^{d-1}$ where $d = \deg(g)$. In fact, for any $f\in A[X]$, it is easy to see that there is a $h\in A[X]$ of $\deg(h) < d$, hence a $A$-linear combination of $1,X,\ldots,X^{d-1}$, such that $f\equiv h \ \operatorname{mod}(g)$ by a division algorithm. 

Reviewer: Please put each sentence on a new line.

  One has $B \cap K = A$.
\end{lemma}
\begin{proof}

>   $A\subset B\cap K$ is trivial. For the other direction, we have $B\cap K = \{x\in K\colon x \text{ is integral over } A\} = A$. The second equality is the definition of normality. 

Reviewer: Sentences should not start with a formula.
```suggestion
  The inclusion $A\subset B\cap K$ is trivial. 
  For the other direction, we have $B\cap K = \{x\in K\colon x \text{ is integral over } A\} = A$. 
  The second equality is the definition of normality. 
```

  Let $t \in L$ be given.
  As $L|K$ is finite, hence algebraic, there is a monic equation of the form
  \[ t^{n} + w_{1} t^{n-1} + \cdots + w_{n} = 0, \ \ w_{i} \in K. \]

>   Write $w_i = b_i/c_i$ for all $i$ where $b_i, c_i\in A$. We have $t^n + \sum_{i=1}^n t^{n-i}b_i/c_i = 0$. Multiply both sides by $\prod_i c_i$, we obtain an equation of the form

Reviewer: Each sentence on a new line please.

  \[ a_{n} t^{n} + \cdots + a_{1} t + a_{0} = 0, \ \ a_{i} \in A. \]

>   Multiply again by $a_{n}^{n-1}$ to obtain an integral equation for $a_{n} t$, namely \[(a_n t)^n + a_{n-1}(a_nt)^{n-1} +\cdots + a_1a_n^{n-2}(a_nt) + a_n^{n-1}a_0 = 0,\] which implies that $a_{n} t \in B$. 

Reviewer: Please put displayed formulas on a new line, as follows. Make sure lines do not end with a space.
```suggestion
  Multiply again by $a_{n}^{n-1}$ to obtain an integral equation for $a_{n} t$, namely 
  \[(a_n t)^n + a_{n-1}(a_nt)^{n-1} +\cdots + a_1a_n^{n-2}(a_nt) + a_n^{n-1}a_0 = 0,\]
  which implies that $a_{n} t \in B$.
```
